main/templates/main/index.html

```diff
@@ -0,0 +1,5 @@
+{% extends "main\base.html" %}
```

Comment:
This will most likely result in an error on unix machines because of the backslash.